### ISSUE TITLE
Moosh creates a course-backup in current directory and inside of the sitedata in course backup area

### ISSUE THREAD

**Issue by CG-CoSector** [tmuras/moosh#453]
- moosh version: 1.11
- moodle version: 4.1.3
    - database: Percona MySQL 5.7.43
- php version: 7.4.3
- operating system: Ubuntu 20.04.6

### Actual behaviour 
Creates a backup file in the current directory **and** a copy backup inside of the sitedata in the course backup file area (duplicating storage used as it's done twice)

### Expected behaviour 
Only creates a single backup file in the current directory

### Steps to reproduce 
Use the course-backup functionality to backup a course id - then check your mdl_files table for a backup file called backup.mbz which should be for the course you just backed up.

**Comment by niallfleming** [comment]
yes, having exactly this issue too. The built in tool from moodle does the same thing but it does delete it after, it's a pain because I'm trying to backup a course that's 40G and don't have the space on the moodledata partitiion to do so, so I had added another disk for the backup to be stored on and fudged the tempdir for moosh only to use that new disk too.

it completes the backup, but bails at storing the file in filedir - prior presumably to moving it where I specified. I see the backup.mbz in the tempdir but don't know if I can trust that as a completed backup file?
